src/Resources/translations/CmfSonataAdminIntegrationBundle.de.xliff

```diff
@@ -287,8 +284,7 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Einstellungen zum Veröffentlichen</target>
```

Comment:
It is mostly the same, but will discuss that with my grammar friend and make it consistent.